qemu-options.hx

```diff
@@ -191,15 +191,15 @@ DEF("numa", HAS_ARG, QEMU_OPTION_numa,
     QEMU_ARCH_ALL)
 SRST
 ``-numa node[,mem=size][,cpus=firstcpu[-lastcpu]][,nodeid=node][,initiator=initiator]``
-  \ 
```

Comment:
Probably best to avoid this diff, even though removing newlines is generally good, I'd avoid any extra source of conflict with upstream